checkstyle/suppressions.xml

```diff
@@ -365,10 +365,11 @@
     <suppress checks="MagicNumber" files="com/hazelcast/util/QuickMath"/>
     <suppress checks="NPathComplexity|CyclomaticComplexity" files="com/hazelcast/client/util/ClientDelegatingFuture"/>
     <suppress checks="IllegalImport|InnerAssignment" files="com/hazelcast/internal/util/counters/SwCounter"/>
-    <suppress checks="JavadocMethod" files="com/hazelcast/util/executor/"/>
-    <suppress checks="JavadocType" files="com/hazelcast/util/executor/"/>
-    <suppress checks="JavadocVariable" files="com/hazelcast/util/executor/"/>
-    <suppress checks="NPathComplexity" files="com/hazelcast/util/executor/DelegatingFuture"/>
+    <suppress checks="JavadocMethod" files="com/hazelcast/internal/util/executor/"/>
```

Comment:
No JavaDoc suppressions for /internal/ packages are needed, there is already a generic filter at the beginning of the file.